maestro-orchestra/src/test/java/maestro/orchestra/yaml/YamlCommandReaderTest.kt

```diff
@@ -5,45 +5,9 @@ import java.nio.file.FileSystems
 import java.nio.file.Paths
 import maestro.KeyCode
 import maestro.ScrollDirection
+import maestro.SwipeDirection
 import maestro.TapRepeat
-import maestro.orchestra.ApplyConfigurationCommand
-import maestro.orchestra.AssertConditionCommand
-import maestro.orchestra.BackPressCommand
-import maestro.orchestra.ClearKeychainCommand
-import maestro.orchestra.ClearStateCommand
-import maestro.orchestra.Command
-import maestro.orchestra.Condition
-import maestro.orchestra.CopyTextFromCommand
-import maestro.orchestra.ElementSelector
-import maestro.orchestra.EraseTextCommand
-import maestro.orchestra.EvalScriptCommand
-import maestro.orchestra.HideKeyboardCommand
-import maestro.orchestra.InputRandomCommand
-import maestro.orchestra.InputRandomType
-import maestro.orchestra.InputTextCommand
-import maestro.orchestra.LaunchAppCommand
-import maestro.orchestra.MaestroCommand
-import maestro.orchestra.MaestroConfig
-import maestro.orchestra.MaestroOnFlowComplete
-import maestro.orchestra.MaestroOnFlowStart
-import maestro.orchestra.OpenLinkCommand
-import maestro.orchestra.PasteTextCommand
-import maestro.orchestra.PressKeyCommand
-import maestro.orchestra.RepeatCommand
-import maestro.orchestra.RunFlowCommand
-import maestro.orchestra.RunScriptCommand
-import maestro.orchestra.ScrollCommand
-import maestro.orchestra.ScrollUntilVisibleCommand
-import maestro.orchestra.SetLocationCommand
-import maestro.orchestra.StartRecordingCommand
-import maestro.orchestra.StopAppCommand
-import maestro.orchestra.KillAppCommand
-import maestro.orchestra.StopRecordingCommand
-import maestro.orchestra.TakeScreenshotCommand
-import maestro.orchestra.TapOnElementCommand
-import maestro.orchestra.TapOnPointV2Command
-import maestro.orchestra.TravelCommand
-import maestro.orchestra.WaitForAnimationToEndCommand
+import maestro.orchestra.*
```

Comment:
Can you fix this wildcard, we try to avoid wildcard imports?

Reply:
Done